src/unk_02070428.c

```diff
@@ -32,28 +32,28 @@ void sub_02070428 (FieldSystem * fieldSystem, BOOL param1)
     fieldSystem->unk_B8 = param1;
 }
 
-void sub_02070430 (FieldSystem * fieldSystem)
+void FieldSystem_InitFlagsNoWarp (FieldSystem * fieldSystem)
```

Comment:
Is this what initializes the overworld NPCs? If so, I'm not sure that I follow this name.

Reply:
Loads all the flags of the area when the map changes via not a warp (think Sangem Town to route 201)

Reply:
Open to better names

Reply:
`FieldSystem_InitNextMapFlags`? `FieldSystem_InitFlagsOnMapChange`?

And then use `Warp` as a suffix to delineate it from this function (though it could be, e.g., `FieldSystem_InitFlagsOnMapWarp`).

To me, a `map change` denotes a smooth flow from one map to the next along a map-matrix, and a `warp` denotes a change across matrices.

Reply:
The second sounds good, but then I'll need to change a few other function names. Nbd tho as long as it increases clarity